windows/src/engine/keyman32/hookutils.cpp

```diff
@@ -15,45 +15,44 @@ typedef BOOL
     IN CONST PMINIDUMP_CALLBACK_INFORMATION CallbackParam OPTIONAL
     );
 
-
-DWORD ExceptionMessage(LPSTR Proc, LPEXCEPTION_POINTERS ep)
-{
+DWORD ExceptionMessage(LPSTR Proc, LPEXCEPTION_POINTERS ep) {
```

Comment:
For this function, I opted to use lower level debugging messages because we are in an exception state.